modules/plot/main/coffee/Axis.coffee

```diff
@@ -31,6 +41,7 @@ dimension = (axis, options) ->
         state[name]
 
   {
+    doCollisionDetection: setterGetter('doCollisionDetection'),
```

Comment:
Charlie suggested the user could still need to see all labels (eg if they are angled).

But we do not want to use angled graph labels for other reasons.